core/src/test/scala/unit/kafka/server/HighwatermarkPersistenceTest.scala

```diff
@@ -47,9 +50,11 @@ class HighwatermarkPersistenceTest {
   @Test
   def testHighWatermarkPersistenceSinglePartition() {
     // mock zkclient
-    val zkUtils = EasyMock.createMock(classOf[ZkUtils])
-    EasyMock.replay(zkUtils)
-    
+    val zkClient = EasyMock.createNiceMock(classOf[ZkClient])
```

Comment:
This is left over from changes required when leveraging zookeeper to track deletes. I left it because it looks like its closer to what was actually trying to be mocked based on the comment. I can keep it or leave it.